### DISCUSSION TITLE
Navigation Shortcuts

### DISCUSSION THREAD

**Discussion by ghost** [spacedriveapp/spacedrive#1892] [Feature Request]
Like typing m in the folders will highlight all folders starting with that to like Windows file explorer does it or a better version where if i typing m and then a it'll show me everything starting with ma like in directory opus and also stuff like backspace like going back to the previous folder

**Comment by utkubakir** [comment]
Thanks for the suggestion @AyaanMAG , this is already in our to-do list, hopefully it will come out in the future updates.

**Reply by ghost** [reply]
Thank you!
